build.assets/Dockerfile

```diff
@@ -245,7 +245,7 @@ RUN export NODE_ARCH=$(if [ "$BUILDARCH" = "amd64" ]; then echo "x64"; else echo
     mkdir -p ${NODE_PATH} && \
     curl -o /tmp/nodejs.tar.xz -fsSL ${NODE_URL} && \
     tar -xJf /tmp/nodejs.tar.xz -C /usr/local/lib/nodejs-linux --strip-components=1
-RUN npm install -g corepack@0.31.0 && corepack enable yarn pnpm
+RUN corepack enable yarn pnpm
```

Comment:
Do we still need yarn here?

Reply:
`build.assets/Makefile` still seems to use it in this single place:

https://github.com/gravitational/teleport/blob/6190627ef77121ee512a1beeab5e1c48ffb93e50/build.assets/Makefile#L705-L708